.travis.yml

```diff
@@ -7,11 +7,14 @@ cache:
     - $HOME/.composer/cache
 
 php:
-  - 5.6
   - 7.0
   - 7.1
   - hhvm
 
+matrix:
+    allow_failures:
+        - php: hhvm
```

Comment:
This is not a solution: check if a newer HHVM version is available - HHVM now support PHP7 stuff.